public/css/style.css

```diff
@@ -1,57 +1,21 @@
-body {
-  padding: 50px;
-  font: 16px "Lucida Grande", Helvetica, Arial, sans-serif;
+* {
```

Comment:
Would it be possible to include this file in all other `CSS` files, and save all colours in this general style file using CSS variables? (Like some of us did in DiscardHumanity).